python/mlc_chat/interface/compile.py

```diff
@@ -102,6 +104,23 @@ def _get_param_metadata(name: str, param: nn.Parameter) -> Dict[str, Any]:
             "preprocs": param.attrs["preprocs"],
         }
 
+    def _find_kv_cache_bytes(model: nn.Module, model_config) -> int:
+        all_kv_cache = nn.core._attribute_finder(  # pylint: disable=protected-access
+            model,
+            prefix="",
+            condition_yield=lambda x: isinstance(x, nn.KVCache),
```

Comment:
@MasterJH5574 This for now only assumes `nn.KVCache`, and we will need to update this method to support serving cases where KVCache can be more complicated.